### PR TITLE
 Post-merge fixes for Course model and data loader

### PR DESCRIPTION
**Pull request by clintonb**
This adds an `openedx.yaml` file, as described by OEP-2:
http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html

The data in this file was transformed from the contents of
edx/repo-tools-data:repos.yaml

### CODE REVIEW

**Comment by clintonb** [comment]
FYI @edx/ecommerce

File: course_discovery/apps/course_metadata/migrations/0022_remove_duplicate_courses.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def remove_duplicate_courses(apps, schema_editor):
+    Course = apps.get_model('course_metadata', 'Course')
+    duplicates = Course.objects.raw('SELECT'
```
Comment (rlucioni) [inline course_discovery/apps/course_metadata/migrations/0022_remove_duplicate_courses.py:9]: Why the raw SQL?

Comment (clintonb) [inline course_discovery/apps/course_metadata/migrations/0022_remove_duplicate_courses.py:9]: `GROUP BY` is not well-support by Django's ORM.

Comment (rlucioni) [inline course_discovery/apps/course_metadata/migrations/0022_remove_duplicate_courses.py:9]: Doesn't raw SQL seem like it should be a last resort? You could probably achieve the same effect using a combination of `values` and `annotate`.

Comment (clintonb) [inline course_discovery/apps/course_metadata/migrations/0022_remove_duplicate_courses.py:9]: I've investigated. This is the best option.

**Comment by rlucioni** [comment]
👍

**Comment by cpennington** [comment]
@clintonb: Apologies. I had a rogue piece of automation that squashed your PR title and message.

**Comment by clintonb** [comment]
No worries @cpennington

**Pull request merged**